README.md

```diff
@@ -23,7 +23,9 @@ server6:
         # - dns: 8.8.8.8 8.8.4.4 2001:4860:4860::8888 2001:4860:4860::8844
 
 #server4:
-#    listen: '127.0.0.1:67'
+#    listen: '0.0.0.0:67'
+#    plugins:
+         # - IPv4: leases4.txt 10.10.10.1 255.255.255.0 8.8.8.8 60 255.255.255.0
```

Comment:
How about using `10.10.10.0/24` here instead of `10.10.10.1 255.255.255.0`.

Then you can use https://golang.org/pkg/net/#ParseCIDR

Reply:
I think this example is out of date.